walk-in-interview/src/main/webapp/new-post.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+
+<body>
+
+    <form>
+        <div class="header-container">
+            <button type="reset" onclick="location.href='/walk-in-interview/src/main/webapp/index.html'">Cancel</button>
+            <div>New Job Post</div>
+            <input type="submit" value="Post">
+        </div>
+        <div class="content-container">
+            <input type="text" class="job-field" id="new-job-title" name="job-title" placeholder="Job Title" />
+            <textarea class="job-field" id="new-job-description" name="job-description"
+                placeholder="Describe the job here (responsibilities, preffered skills)..."></textarea>
+            <input type="text" class="job-field" id="new-job-address" name="job-address" placeholder="Job Address" />
+            
+            <div class="job-field">
+                Requirements (for applicants):
+                <input type="checkbox" id="o-level" name="o-level" value="o-level">
+                <label for="o-level">O Levels</label>
+                <input type="checkbox" id="drivers-license" name="drivers-license" value="drivers-license">
+                <label for="drivers-license">Driver's License</label>
+            </div>
```

Comment:
I think we need to specify the requirements list:) personally I am not familiar with credentials in Singapore

Reply:
I think for now we can just add in a few example requirements, and write the code such that it is easy to add/delete requirements (I think this was the plan anyways). So this means the HTML select options would be rendered dynamically